<!DOCTYPE HTML>
<!--
	TXT by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Research </title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
	</head>
	<body class="is-preload">
		<div id="page-wrapper">

			<!-- Header -->
				<header id="header">
					<div class="logo container">
						<div>
							<h1><a href="index.html" id="logo">ComBi</a></h1>
							<p> Reserach Topics </p>
						</div>
					</div>
				</header>

			<!-- Nav -->
				<nav id="nav">
					<ul>
						<li><a href="index.html">Home</a></li>
						<li><a href="people.html">People</a></li>
						<li class="current"><a href="research.html">Research</a></li>
						<li><a href="news.html">News</a></li>
						<li><a href="publications.html">Publications</a></li>
						<li><a href="teaching.html">Teaching</a></li>
					</ul>
				</nav>

			<!-- Main -->
				<section id="main">
					<div class="container">
						<div class="row">
							<div class="col-12">
								<div class="content">

									<!-- Content -->

										<article class="box page-content">
											<header>
												<h2>Research</h2>
												<p> A short overview of our current and past research projects. </p>
											</header>

											<!-- Project: Local Adaptation and Chromosomal Inversions -->
											<section class="box feature" style="margin-bottom:2em;">
												<div class="row aln-middle">
													<div class="col-5 col-12-small">
														<img src="images/Inversions.png" alt="Inversions Research" class="responsive-image" />
													</div>
													<div class="col-7 col-12-small">
														<h3>Local Adapation and Chromosomal Inversions</h3>
														<p>
															Nature showcases a fascinating mosaic of adaptations, with species evolving to adapt to their ever-changing environments. Chromosomal inversions, structural mutations that reverse gene order and link different gene variants, play a key role in this process. These inversions, often influenced by environmental factors like temperature and altitude, contribute to reproductive isolation and new species emergence. However, the exact mechanisms behind their contribution to local adaptation and the formation of new species remain unclear.
														</p>
														<p>
															This project explores whether genomic structure facilitates the establishment of locally adaptive mutations or if natural selection and adaptive genome content drive the evolution of structural variation. By developing new statistical methods and mathematical models, and using machine learning to analyze data from sticklebacks and fruit flies, the project aims to demonstrate how chromosomal inversions contribute to local adaptation.
														</p>
														<p>
															<b>Team:</b> Stefan Struett, Karolina Wachala, Stephan Peischl<br>
															<b>Project Partners:</b> Thomas Flatt, Katie Peichel 
														</p>
														<p>
															Supported by SNSF grant 10001034 (<a href="https://data.snf.ch/grants/grant/10001034" target="_blank">SNSF Link</a>)
														</p>
														
													</div>
												</div>
											</section>
											<hr style="margin:2em 0;">

											<!-- Project: Automated Machine Learning -->
											<section class="box feature" style="margin-bottom:2em;">
												<div class="row aln-middle">
													<div class="col-5 col-12-small">
														<img src="images/EVR_project.png" alt="Automated Machine Learning Project" class="responsive-image" />
													</div>
													<div class="col-7 col-12-small">
														<h3>Automated Machine learning to identify drivers of microbial community composition</h3>
														<p>
															The compositions of microbial communities are of central importance for ecosystem functioning and host-associated health. Recent advancements in metagenomic sequencing have facilitated the acquisition of microbiome composition data. However, current analytical methods often fall short in exploring the environmental factors shaping these communities, especially in complex settings with numerous variables.
														</p>
														<p>
															Given the crucial ecologic role of microbiomes and methodological gaps, this project aims to develop an analytic pipeline that explores environmental variables and ranks them by their association with the microbial community compositions at hand. The developed pipeline implements best practices of conditional data analysis (CoDa) and uses state-of-the-art ordination, clustering, and machine learning techniques to handle data with minimal supervision and assumptions.
														</p>
														<p>
															<b>Team:</b> Ianis Vilela, Stephan Peischl<br>
															<b>Co-Supervisors:</b> Claudia Bank, Adamantia Kapopoulou
														</p>
														<p>

													</div>
												</div>
											</section>
											<hr style="margin:2em 0;">

											<!-- Project: Evolutionary Rescue -->
											<section class="box feature" style="margin-bottom:2em;">
												<div class="row aln-middle">
													<div class="col-5 col-12-small">
														<img src="images/EvoRescue.jpeg" alt="Evolutionary Rescue Research" class="responsive-image" />
													</div>
													<div class="col-7 col-12-small">
														<h3>Evolutionary Rescue</h3>
														<p>
															We study mathematical models of evolutionary rescue in tempoprally and spatially changing extended populations.
														</p>
														<p>
															<b>Team:</b> Matteo Tomasini, Stephan Peischl
														</p>
														<p>
															<b>Key publication:</b>
															<a href="https://doi.org/10.1111/evo.14038" target="_blank">
																When does gene flow facilitate evolutionary rescue?
															</a>
														</p>
													</div>
												</div>
											</section>
											<hr style="margin:2em 0;">

											<!-- Project: Range Expansions -->
											<section class="box feature" style="margin-bottom:2em;">
												<div class="row aln-middle">
													<div class="col-5 col-12-small">
														<img src="images/RangeExpansion.png" alt="RE Research" class="responsive-image" />
													</div>
													<div class="col-7 col-12-small">
														<h3>Range Expansions</h3>
														<p>
															Range expansions have profound effects on neutral and functional genomic diversity. 
															We study the impact of gene surfing on the build up of expansion load and the evolution of fintess and dispersal related traits during range expansions.
														</p>
														<p>
															<b>Team:</b> Stephan Peischl<br>
															<b>Collaborators:</b> Laurent Excoffier, Kim Gilbert, Mark Kirkpatrick and others
														</p>
														<p>
															<b>Key publication:</b>
															<a href="https://doi.org/10.1111/mec.12524" target="_blank">
																On the accumulation of deleterious mutations during range expansions
															</a>
														</p>
													</div>
												</div>
											</section>
											<hr style="margin:2em 0;">

											<!-- Project: Evolution in regions of low recombination -->
											<section class="box feature" style="margin-bottom:2em;">
												<div class="row aln-middle">
													<div class="col-5 col-12-small">
														<img src="images/ESC_model.png" alt="Evolution in Regions of Low Recombination" class="responsive-image" />
													</div>
													<div class="col-7 col-12-small">
														<h3>Evolution in regions of low recombination</h3>
														<p>
															We study how deleterious mutations can affect patterns of diversity and demographic inference in regions of low recombination.
														</p>
														<p>
															<b>Team:</b> Stefan Struett, Stephan Peischl<br>
															<b>Collaborator:</b> Laurent Excoffier
														</p>
														<p>
															<b>Key publication:</b>
															<a href="https://doi.org/10.1093/genetics/iyaf013" target="_blank">
																A generalized structured coalescent for purifying selection without recombination
															</a>
														</p>
													</div>
												</div>
											</section>
											<hr style="margin:2em 0;">

											<!-- Project: Teaching mathematical modelling with interactive web apps -->
											<section class="box feature" style="margin-bottom:2em;">
												<div class="row aln-middle">
													<div class="col-5 col-12-small">
														<img src="images/Hermina_research.gif" alt="Shiny Apps Research" class="responsive-image" />
													</div>
													<div class="col-7 col-12-small">
														<h3>Teaching mathematical modelling with interactive web apps</h3>
														<p>
															R Shiny apps can improve the dissemination of mathematical models in ecology and evolution. We study how these interactive tools can be used most effectively in higher-education.
														</p>
														<p>
															<b>Team:</b> Ana-Hermina Ghenu & Stephan Peischl<br>
															<b>Project Partner:</b> Sandra Grinschgl
														</p>
														
						
													</div>
												</div>
											</section>
											<hr style="margin:2em 0;">

											<!-- Project: Mutation Load Dynamics -->
											<section class="box feature" style="margin-bottom:2em;">
												<div class="row aln-middle">
													<div class="col-5 col-12-small">
														<img src="images/Vole_Load.jpg" alt="Mutation Load Dynamics" class="responsive-image" />
													</div>
													<div class="col-7 col-12-small">
														<h3>Mutation Load Dynamics</h3>
														<p>
															We study when and how deleterious mutations can accumulate in genomes. This includes investigating the demographic and evolutionary processes that shape the distribution and effects of deleterious mutations in natural populations.
														</p>
														<p>
															<b>Team:</b> Stephan Peischl<br>
															<b>Collaborators:</b> Gerald Heckel, Laurent Excoffier, Flavia Schlichta, Kim Gilbert and others
														</p>
														<p>
															<b>Key publication:</b>
															<a href="https://www.cell.com/current-biology/fulltext/S0960-9822(23)00524-9" target="_blank">
																Demographic history and genomic consequences of 10,000 generations of isolation in a wild mammal
															</a>
														</p>
													</div>
												</div>
											</section>

										</article>

								</div>
							</div>
						</div>
					</div>
				</section>

			<!-- Footer -->
				<footer id="footer">
					<div class="container">
						<div class="row gtr-200">
							<div class="col-12">
								<!-- "What's this about?" section removed -->
							</div>
						</div>

						<!-- Copyright -->
							<div id="copyright">
								<ul class="menu">
									<li>Design: <a href="http://html5up.net">HTML5 UP</a> by <a href="http://twitter.com/ajlkn">AJ</a></li>
								</ul>
							</div>

					</div>
				</footer>

		</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.dropotron.min.js"></script>
			<script src="assets/js/jquery.scrolly.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>
